code/modules/vore/fluffstuff/custom_items_vr.dm

```diff
@@ -891,7 +891,7 @@
 	set src in view(1)
 
 	//do_reagent_implant(usr)
-	if(!isliving(usr) || !usr.checkClickCooldown())
+	if(!isliving(usr) || !usr.canClick())
```

Comment:
Nop

Reply:
Outdated branch?

Reply:
whoops yep outdated branch
lemme fix that